src/main/java/org/broadinstitute/hellbender/tools/exome/orientationbiasvariantfilter/OrientationBiasFilterer.java

```diff
@@ -58,7 +58,7 @@ public static VariantContext annotateVariantContextWithPreprocessingValues(final
 
             final List<Allele> alleles = genotype.getAlleles();
             if (genotype.getPloidy() != 2) {
-                logger.warn("No action required:  This tool will skip non-diploid sites.  Saw GT: " + genotype.getGenotypeString() + " at " + vc.toStringWithoutGenotypes());
+                logger.debug("No action required:  This tool will skip non-diploid sites.  Saw GT: " + genotype.getGenotypeString() + " at " + vc.toStringWithoutGenotypes());
```

Comment:
Maybe use the `{}` syntax of the logger to avoid the construction of the genotype/variant strings:

```java
logger.debug("No action required:  This tool will skip non-diploid sites.  Saw GT: {} at {}",
    genotype::getGenotypeString, vc::toStringWithoutGenotypes);
```